client/lib/abtest/active-tests.js

```diff
@@ -82,6 +82,15 @@ module.exports = {
 		defaultVariation: 'designTypeWithoutStore',
 		allowExistingUsers: false,
 	},
+	signupThemePreview: {
+		datestamp: '20160727',
+		variations: {
+			showThemePreview: 20,
+			hideThemePreview: 80,
+		},
+		defaultVariation: 'hideThemePreview',
+		allowExistingUsers: true,
```

Comment:
`allowExistingUsers` defaults to true in [abtest/index.js](https://github.com/Automattic/wp-calypso/blob/master/client/lib/abtest/index.js#L105)